partitioned-table.md

```diff
@@ -560,6 +560,10 @@ MOD(YEAR('2005-09-01'),4)
 =  1
 ```
 
+#### LINEAR HASH handling
+
+`PARTITION BY LINEAR HASH` is accepted by the LINEAR is ignored, and it will be created as a non-linear HASH partitioned table. Before v6.4 it was created as a non-partitioned table.
```

Comment:
“`PARTITION BY LINEAR HASH` is accepted by the LINEAR is ignored”: there are two "is" in this sentence, what does it mean? @mjonss Would you please update it? Thanks.

Reply:
@qiancai Thank you for the review, I have updated the phrasing, please take a look.